engine/linter/testdata/volume_claim.yml

```diff
@@ -0,0 +1,36 @@
+kind: pipeline
+type: kubernetes
+name: default
+
+clone:
+  disable: true
+
+steps:
+- name: write
+  pull: if-not-exists
+  image: alpine
+  volumes:
+  - name: shared
+    path: /shared
+  commands:
+  - pwd
+  - ls /shared
+
+- name: read
+  pull: if-not-exists
+  image: alpine
+  volumes:
+  - name: shared
+    path: /shared
+  commands:
+  - pwd
+  - ls /shared
+  - sleep 500
+
+# once the pipeline finished, check the host machine
+# to see if exists /tmp/drone/test/greetings.txt
+
+volumes:
+- name: shared
+  claim:
+    claimName: received-data-claim
```

Comment:
we use snake case to name our attributes, however, in this case we can simplify this from `claimName` to just `name`